src/data-loaders/bookmark-widget.js

```diff
@@ -13,8 +13,8 @@ const _ = {
 const reduxStatePropKey = twreporterRedux.reduxStateFields
 
 const host = {
-  master: 'http://testtest.twreporter.org:3000',
-  preview: 'http://testtest.twreporter.org:3000',
```

Comment:
`preview` branch is for keystone preview
I think we could keep it since it's still under maintenance

Reply:
fixed